Makefile

```diff
@@ -190,14 +190,13 @@ else
  NIM_EXTRA_PARAMS := --passL:"-lsetupapi -lhid"
 endif
 
-
-ifeq ($(detected_OS),Darwin)
-  ifeq ("$(shell sysctl -nq hw.optional.arm64)","1")
-    # Building on M1 is still not supported, so in the meantime we crosscompile to amd64
-    DOTHERSIDE_CMAKE_PARAMS += -DCMAKE_OSX_ARCHITECTURES=x86_64
-	NIM_PARAMS += --cpu:amd64 --os:MacOSX --passL:"-arch x86_64" --passC:"-arch x86_64"
-  endif
-endif
+# ifeq ($(detected_OS),Darwin)
+#   ifeq ("$(shell sysctl -nq hw.optional.arm64)","1")
+#     # Building on M1 is still not supported, so in the meantime we crosscompile to amd64
+#     DOTHERSIDE_CMAKE_PARAMS += -DCMAKE_OSX_ARCHITECTURES=x86_64
+# 	NIM_PARAMS += --cpu:amd64 --os:MacOSX --passL:"-arch x86_64" --passC:"-arch x86_64"
+#   endif
+# endif
```

Comment:
Are you sure? We don't have Qt 5.15.x for arm64 on CI

Reply:
It was just a quick test and pushed with the hope of helping @alexjba to check the performance improvements running native on M1